rcl_logging_spdlog/test/fixtures.hpp

```diff
@@ -109,15 +109,22 @@ class LoggingTest : public AllocatorTest
   }
 
 private:
-  std::string get_expected_log_prefix()
+  std::string get_expected_log_prefix(const char * name)
   {
-    char * exe_name = rcutils_get_executable_name(allocator);
-    if (nullptr == exe_name) {
-      throw std::runtime_error("Failed to determine executable name");
+    char * exe_name;
+    if (name == nullptr || name[0] == '\0') {
+      exe_name = rcutils_get_executable_name(allocator);
+      if (nullptr == exe_name) {
+        throw std::runtime_error("Failed to determine executable name");
+      }
+    } else {
+      exe_name = const_cast<char *>(name);
```

Comment:
Same here; it is just easier to do the duplication, and then unconditionally free later on.

Reply:
addressed with https://github.com/ros2/rcl_logging/pull/109/commits/1c6dffc12b6d9342b04cbd072fcf54aab63f918f